<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hand_3DClass</class>
 <widget class="QMainWindow" name="Hand_3DClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hand_3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="MyOpenGLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopenFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopenFile"/>
   </widget>
   <addaction name="menuopenFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFront"/>
   <addaction name="actionSlide"/>
   <addaction name="actionTop"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFront">
   <property name="text">
    <string>主视图</string>
   </property>
  </action>
  <action name="actionSlide">
   <property name="text">
    <string>侧视图</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>俯视图</string>
   </property>
  </action>
  <action name="actionopenFile">
   <property name="text">
    <string>openFile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">myopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Hand_3D.qrc"/>
 </resources>
 <connections/>
</ui>
